WTWSMS/history/provinces/676 - Kakheti.txt

```diff
@@ -15,8 +15,12 @@ b_davidgareja = temple
 #b_gremi = castle
 
 # Misc
-culture = armenian
-religion = georgian_orthodox
+culture = juhuro
+religion = tzadikite
```

Comment:
They weren't Armenian though, as culture in this area tended to be associated to religion.